app/controllers/hq/partners_controller.rb

```diff
@@ -6,6 +6,19 @@ def index
     @partners= Partner.all
   end
 
+  def new
+    @partner = Partner.new
```

Comment:
Yeah, we were wondering whether we should use the `build_partner` method or just a normal Partner.new. I couldn't see what would be the benefit of running the extra line in `build_partner` when we are sure this line is not going to do anything.